src/game/default/g_ballistics.c

```diff
@@ -315,6 +315,69 @@ static void G_GrenadeProjectile_Explode(g_entity_t *self) {
 			G_BurnMark(self->s.origin, plane, surf, 20);
 		}
 	}
+	
+	// attempt to stop the sound playing...doesnt work
+	self->s.sound = 0;
```

Comment:
This might work if you set self->locals.next_think to g_level.time + frame_millis, and set self->locals.Think to G_FreeEdict. That way, the sound update will be sent to the client.